src/process_utility.c

```diff
@@ -3374,6 +3374,41 @@ process_altertable_chunk_set_tablespace(AlterTableCmd *cmd, Oid relid)
 	}
 }
 
+/*
+ * Set the access method for a table.
+ *
+ * If called on a hypertable, this will set the compression flag on the
+ * hypertable in addition to running the set access method code.
+ */
+#if PG15_GE
+static void
+process_set_access_method(AlterTableCmd *cmd, ProcessUtilityArgs *args)
+{
+	AlterTableStmt *stmt = castNode(AlterTableStmt, args->parsetree);
+	Oid relid = AlterTableLookupRelation(stmt, NoLock);
+	Cache *hcache;
+	Hypertable *ht = ts_hypertable_cache_get_cache_and_entry(relid, CACHE_FLAG_MISSING_OK, &hcache);
+	if (ht && (strcmp(cmd->name, "hyperstore") == 0))
```

Comment:
This should be changed to the correct name

Reply:
Is that `hypercore`?

Reply:
Used `hypercore_tam` as you suggested.